types/bunyan/bunyan-tests.ts

```diff
@@ -129,3 +129,6 @@ class MyLogger extends Logger {
         super(options);
     }
 }
+
+const nameLevel: number = Logger.levelFromName['warn'];
```

Comment:
If you make the other change I proposed, I might suggest also changing this test to use a variable of type `LogLevelString` instead of just `'warn'`